scripts/archive.coffee

```diff
@@ -26,6 +26,8 @@ config =
   github_token: process.env.HUBOT_GITHUB_ACCESS_TOKEN
   archive_repo: process.env.HUBOT_ARCHIVE_REPO
 
+[config.archive_repo_login, config.archive_repo_name] = config.archive_repo.split '/'
```

Comment:
dumbbbbbb question, the config is from the heroku app and there is more in there than the .env yeah? Is there anywhere the config stuff is documented in this repo to be explicit?

Reply:
Ah no, good point. The hubot way is normally to doc most stuff in the header of the `scripts/myscript.coffee` file, in the "configuration" or "notes" section:
https://github.com/hyphacoop/hyphacoop-chatbot/pull/13/files#diff-5fb33b9df24ae6b0ded9c438d55101ccL5

But that's not pointed to. I was trying to do more high-level human-friendly docs in README -- where someone not deeply technical would think to look.

What if I fleshed out the inline docs to explain envvars config, and ensured readme pointed to file to "technical docs"? Open to an alternative, but i'll make a quick change to demo. Easy to move it elsewhere!

Reply:
Oh wait. You're just wondering why there's more in the .env?

That's for some pieces that aren't custom scripts. For example, the chat adapter for matrix, and an outside script to help app stay alive on heroku, which is referenced in https://github.com/hyphacoop/hyphacoop-chatbot#deployment

If we annotated the `sample.env` to hint at that, would that close the loop?

Reply:
```suggestion
[config.archive_repo_user, config.archive_repo_name] = config.archive_repo.split '/'
```